vcpkg/users/binarycaching.md

```diff
@@ -1,23 +1,57 @@
 ---
+# Concepts/Binary Caching
```

Comment:
It looks like this article should turn into a reference article since consume/binary-caching.md covers the concepts.